src/FluentValidation/Internal/Extensions.cs

```diff
@@ -16,32 +16,40 @@
 // The latest version of this file can be found at https://github.com/jeremyskinner/FluentValidation
 #endregion
 
-namespace FluentValidation.Internal {
+namespace FluentValidation.Internal
+{
 	using System;
 	using System.Collections.Generic;
 	using System.Linq.Expressions;
 	using System.Reflection;
+	using System.Runtime.CompilerServices;
 	using System.Text;
 	using System.Threading;
 	using System.Threading.Tasks;
 
 	/// <summary>
 	/// Useful extensions
 	/// </summary>
-	public static class Extensions {
-		internal static void Guard(this object obj, string message) {
-			if (obj == null) {
-				throw new ArgumentNullException(message);
+	public static class Extensions
+	{

```

Comment:
Same as above - lease preserve the original formatting (braces on end of line) here and for the rest of this file.